client/settings/fraud-protection/style.scss

```diff
@@ -211,28 +211,9 @@
 			border-bottom: 1px solid #e0e0e0;
 			border-top: 0;
 		}
-		&-notice {
```

Comment:
This is just removing the styles that I could recognise as being related to the old notice implementation. Now that the notices are handled by the `BannerNotice`, they are no longer needed here.

Reply:
Yep, that'll be better, thank you!